http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -298,24 +297,18 @@ if [ ! -z "$JUPYTER_DOCKER_IMAGE" ] ; then
   mkdir -p /work/packages
   chmod a+rwx /work/packages
 
+  JUPYTER_USER_PIP_DIR=$JUPYTER_USER_HOME/.local/lib/python3.7/site-packages
```

Comment:
Can we put this below the comment on L302-303?

It looks like `JUPYTER_USER_PIP_DIR` and `ROOT_USER_PIP_DIR` are used for setting `PIP_TARGET` when installing extensions. However we now set `PIP_TARGET=${NOTEBOOKS_DIR}/packages` in the `docker-compose`. 

I can't remember why we need a different `PIP_TARGET` for extension installation.. 🤔

Reply:
my guess is because we don't want extensions persisted in PD? cuz we'll be installing it during runtime creation again if it exists in PD already

Reply:
yeah that was the reason @Qi77Qi - see https://github.com/DataBiosphere/leonardo/pull/1499. because we set `PIP_TARGET` in the `docker-compose`, the extensions were being installed to that directory (which is the directory that persists) but we don't want these extensions to persist since we install them every time we create a GCE VM